### PR TITLE
modify the log level of some errors

### PR DESCRIPTION
**Pull request by googs1025**
- modify the log level of some errors
- fix method comment

### CODE REVIEW

**Review by elezar — APPROVED** [review]

File: internal/plugin/server.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func (plugin *NvidiaDevicePlugin) Start() error {
 		// TODO: add MPS health check
 		err := plugin.rm.CheckHealth(plugin.stop, plugin.health)
 		if err != nil {
-			klog.Infof("Failed to start health check: %v; continuing with health checks disabled", err)
+			klog.Errorf("Failed to start health check: %v; continuing with health checks disabled", err)
```
Comment (elezar) [inline internal/plugin/server.go:160]: This could also be a warning instead, but I don't think it's a blocker.

**Comment by googs1025** [comment]
do we have to fix this ci error?

```bash
Error: internal/resource/cuda-lib.go:60:15: G115: integer overflow conversion int -> uint (gosec)
  	major := uint(version) / 1000
  	             ^
  Error: internal/resource/cuda-lib.go:61:15: G115: integer overflow conversion int -> uint (gosec)
  	minor := uint(version) % 100 / 10
  	             ^
  Error: internal/resource/nvml-lib.go:44:15: G115: integer overflow conversion int -> uint (gosec)
  	major := uint(v / 1000)
  	             ^
  Error: internal/resource/nvml-mig-device.go:1[31](https://github.com/NVIDIA/k8s-device-plugin/actions/runs/10904087630/job/30259653352?pr=898#step:5:33):16: G115: integer overflow conversion int -> uint64 (gosec)
  		return uint64(totalMemory.(int)), nil

```

File: internal/plugin/server.go (hunk 1 above)
Comment (googs1025) [reply internal/plugin/server.go:160]: Thanks for your reply :)

**Comment by googs1025** [comment]
friendly ping @elezar

**Pull request merged**